cfgov/unprocessed/apps/paying-for-college/js/college-costs/views/navigation-view.js

```diff
@@ -118,35 +119,33 @@ const navigationView = {
       'compare-school': '5',
       'review-plan': '6',
       'action-plan': '7',
-      'save-finish': '8'
+      'save-finish': '8',
     };
-    if ( {}.hasOwnProperty.call(codes, section) ) {
+    if ({}.hasOwnProperty.call(codes, section)) {
       navigationView._navTrack += codes[section] + '-';
     } else {
       navigationView._navTrack += '??-';
     }
   },
 
-
   /**
-   * _timedNavTracking - Fires events after various delays to help track users
+   * _timedNavTracking - Fires events after various delays to help track users.
    */
-  _timedNavTracking: function() {
-    const intervals = [ 60 , 120, 180, 300, 600, 900];
-    intervals.forEach( ( int ) => {
-      setTimeout( () => {
-        const d = new Date().toTimeString();
```

Comment:
This was deleted as unused

Reply:
Whoops, yes, that `const d` was from an earlier iteration and wasn't properly deleted. It can be deleted.